swagger2.0/oic.wk.col.swagger.json

```diff
@@ -268,61 +283,77 @@
     "sbaseline" : {
       "properties": {
         "links" : {
-            "description": "A set of simple or individual OIC Links.",
-            "items": {
+          "description": "A set of simple or individual OIC Links.",
+          "items": {
             "$ref": "#/definitions/oic.oic-link"
-            },
-            "type": "array"
-        },        
-        "n": { "$ref" : "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"},
-        "id": {"$ref" : "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"},
-	      "rt": {	"$ref": "#/definitions/oic.core/properties/rt"},
-	      "rts": { "$ref": "#/definitions/oic.core/properties/rt"},
-	      "rts-m": { "$ref": "#/definitions/oic.core/properties/rt"},
+          },
+          "type": "array"
+        },
+        "n": {
+          "$ref" : "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/n"
+        },
+        "id": {
+          "$ref" : "https://openconnectivityfoundation.github.io/core/schemas/oic.common.properties.core-schema.json#/definitions/id"
+        },
+	      "rt": {
+          "$ref": "#/definitions/oic.core.rt-col"
+        },
+	      "rts": {
+          "$ref": "#/definitions/oic.core.rt"
+        },
+	      "rts-m": {
+          "$ref": "#/definitions/oic.core.rt"
+        },
 	      "if": {
 	        "description": "The interface set supported by this resource",
 	        "items": {
-		        "enum": ["oic.if.baseline",
+		        "enum": [
+              "oic.if.baseline",
```

Comment:
Nit picking, but should the "if" values be lined up?  For some reason oic.if.baseline is not tabbed over as far.

Reply:
Sometimes that happens when things are pushed, I can clean up probably.